### ISSUE TITLE
Suggestion - Disable Issues Section until API is fixed.

### ISSUE THREAD

**Issue by ch333** [AHAAAAAAA/PokemonGo-Map#3466]
Other places have turned off the Issues feature while the API is down.  This should help stop people from posting that their map all of a sudden stopped working and help them actually read other issues and/or the ReadMe section.

**Comment by ericmacfa** [comment]
I've seen 10+ issues raised and closed for the same thing, and I was only watching it for a half hour.

**Comment by toomasio** [comment]
Or a simple sticky at the top of the issues section...if that's possible.

**Comment by DiscoTim** [comment]
should be a notice of this type of issue, so people are aware and don't post these. 
It would be great if there are some form of updates, I've read elsewhere "developers are working on it" but something semi official from the project would be great.

**Comment by DiscoTim** [comment]
I see there is a notice on the front page now, good work!

**Comment by ch333** [comment]
@DiscoTim - The problem is that a lot of people don't even scroll down to see the big bold message.  But if issues were disabled, they would maybe go back and read to see "what's going on."  It seems like 80% of these issues are people too lazy to even use the search function to see if the issue has already been posted, let alone resolved.

**Comment by DiscoTim** [comment]
yeah, anyone posting these doesn't search, and that seems to be a lot of people. it would suck to cut off all communication for everyone. someone else suggested a sticky at the top, not sure if that's possible on github, but its a great idea - atleast cut down on the lay-z posting.
